app/uk/gov/hmrc/customs/notification/controllers/dynamicservice/DynamicServiceConfigurationController.scala

```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.customs.notification.controllers.dynamicservice
 
 import com.google.inject.Inject
-import play.api.libs.json.{JsError, JsSuccess, Json}
+import play.api.libs.json._
```

Comment:
```suggestion
import play.api.libs.json.*
```

Reply:
Tried but it has compile error. Anything that I missed?
<img width="568" alt="image" src="https://github.com/hmrc/customs-notification/assets/132441448/fe302c50-ef09-4c1e-98c2-450998e696d1">